assembl/graphql/schema.py

```diff
@@ -511,6 +515,13 @@ class QuestionInput(graphene.InputObjectType):
     title_entries = graphene.List(LangStringEntryInput, required=True)
 
 
+# How the file upload works
+# With the https://github.com/jaydenseric/apollo-upload-client
```

Comment:
I could be wrong, but they do NOT use the `apollo-file-upload` server module. As a result, the uploaded file is not put in a folder temporarily. The GraphQL query passes the `File` object through the request to `graphene`, and we store directly to DB. In V1, we actually did our control in the front-end. So, very good point. This is a security vulnerability that has existed from an API's perspective for a long time. There is no rate-limiting, no file-size limit, and in fact, no MIME-type limit either. All of these were the next steps in Assembl V1. So perhaps we can add these limits now on the `Document` model (instead of the graphql request) so that both versions (for now) benefit from it.